src/cunumeric/random/bitgenerator_curand.inl

```diff
@@ -0,0 +1,313 @@
+/* Copyright 2021-2022 NVIDIA Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#pragma once
+
+#include <unistd.h>
+#include <sys/syscall.h>
+#include <sys/types.h>
+#include <mutex>
+
+#if 0  // for debugging
```

Comment:
Given that you marked my comments as resolved, I assume you will be removing these custom macros and debugging functions.